cdap-standalone/pom.xml

```diff
@@ -370,33 +370,12 @@
                     </resource>
                     <!-- Copy cdap-ui -->
                     <resource>
-                      <directory>${project.parent.basedir}/cdap-ui/server</directory>
-                      <targetPath>ui/server</targetPath>
-                    </resource>
-                    <resource>
-                      <directory>${project.parent.basedir}/cdap-ui/dist</directory>
-                      <targetPath>ui/dist</targetPath>
-                    </resource>
-		    <resource>
-		      <directory>${project.parent.basedir}/cdap-ui/cdap_dist</directory>
-                      <targetPath>ui/cdap_dist</targetPath>
-		    </resource>
-                    <resource>
-                      <directory>${project.parent.basedir}/cdap-ui/node_modules</directory>
-                      <targetPath>ui/node_modules</targetPath>
-                    </resource>
-                    <resource>
-                      <directory>${project.parent.basedir}/cdap-ui/templates</directory>
-                      <targetPath>ui/templates</targetPath>
-                    </resource>
-                    <resource>
-                      <directory>${project.parent.basedir}/cdap-ui/</directory>
-                      <targetPath>ui/</targetPath>
-                      <includes>
-                        <include>server.js</include>
-                        <include>package.json</include>
-                      </includes>
-                    </resource>
+                     <directory>${project.parent.basedir}/cdap-ui/target/stage-packaging/opt/cdap/ui</directory>
+                     <targetPath>ui</targetPath>
+                     <excludes>
+                      <exclude>**LICENSE**</exclude>
+                     </excludes>
+                   </resource>
```

Comment:
I believe it will only exclude files that have LICENSE in its name. Not sure if that will cause any issue. But built sdk and tested everything before pushing this change.